playbooks/roles/mongodb/defaults/main.yml

```diff
@@ -8,7 +8,7 @@ MONGODB_SERVER_BACKUP_DIR: /var/lib/mongodb/backup
 # PROMETHEUS ##################################################################
 
 MONGODB_EXPORTER_ENABLED: true
-MONGODB_EXPORTER_VERSION: "v0.6.2"
+MONGODB_EXPORTER_VERSION: "v0.20.1"
```

Comment:
@pomegranited I've checked stage and the mongodb exporter isn't working. Probably because of the huge leap in the version here.

Upon reading the docs and testing a few things, I found that we need to make a few changes to make it work again.
I've got `mongodb-exporter` up again with the following parameters:
```
# Notice the double `-` on the parameters.
# `compatible-mode` flag is required so that the exporter 
# logs the same information as the old version and gives us 
# some time before we have to fix our grafana dashboards. 
/usr/local/bin/mongodb_exporter \
  --web.listen-address=":9001" \
  --compatible-mode \
  --mongodb.uri=mongodb://<USERNAME>:<PASSWORD>@localhost:27017/admin
```

I've updated the service file manually on the server to get it back up.

Reply:
Thank you for fixing this @giovannicimolin ! If I could have gotten an earlier version to build, I would have done that, but this was the only one whose `go` dependencies worked.

I've updated the template to use the settings you provided, and redeployed to stage to test.